middleware/draftAppealStoreMiddleware.js

```diff
@@ -124,10 +122,10 @@ class SaveToDraftStoreAddAnother extends AddAnother {
 
   get continueText() {
     if (this.req.idam) {
-      return content.continueButtonText.save;
+      return 'saveAndContinue';
```

Comment:
this hardcoded value, can we please refer from json file

Reply:
That hardcoded value represents the KEY in the json file for the VALUE to use in the template.
Makes sense?